activities/MindMath.activity/index.html

```diff
@@ -88,41 +88,41 @@
 			<sugar-toolitem v-else v-on:click="unfullscreen" id="unfullscreen-button" v-bind:title="l10n.stringUnfullscreen" class="pull-right"></sugar-toolitem>
 			<sugar-toolitem id="help-button" v-bind:title="l10n.stringTutorial" class="pull-right" v-on:click="onHelp"></sugar-toolitem>
 		</sugar-toolbar>
-
-		<transition name="fade" id="app-content" mode="out-in">
-			<component
-				v-bind:is="currentScreen"
-				v-bind:time="clock.time"
-				v-bind:stroke-color="strokeColor"
-				v-bind:fill-color="fillColor"
-				v-bind:questions="questions"
-				v-bind:q-no="qNo"
-				v-bind:score="score"
-				v-bind:mode="mode"
-				v-bind:compulsory-ops="compulsoryOps"
-				v-bind:compulsory-ops-for-each-question="compulsoryOpsForEachQuestion"
-				v-bind:compulsory-ops-rem="compulsoryOpsRem"
-				v-bind:slots="slots"
-				v-bind:scores="scores"
-				v-bind:time-taken="timeTaken"
-				v-bind:input-numbers="inputNumbers"
-				v-bind:input-numbers-types="inputNumbersTypes"
-				v-bind:players-all="playersAll"
-				v-bind:players-playing="playersPlaying"
-				v-bind:disabled="disabled"
-				v-bind:multiplayer="multiplayer"
-				v-bind:sugar-presence="SugarPresence"
-				v-bind:is-target-acheived="isTargetAcheived"
-				v-bind:l10n="l10n"
-				v-on:validate="onValidate"
-				v-on:slots-update="onSlotsUpdated"
-				v-on:restart-game="handleRestartButton"
-				v-on:pass-question="handlePassButton"
-				v-on:see-leaderboard="currentScreen = 'leaderboard'"
-				v-on:go-to-result="currentScreen = 'result'"
-			></component>
-	</transition>
-
+		<div id="app-content">
+			<transition name="fade" id="app-content" mode="out-in">
```

Comment:
remove `id` from `<transition>` i.e. change it to `<transition name="fade" mode="out-in">`